app/controllers/mobile_app/sensors_controller.rb

```diff
@@ -2,7 +2,16 @@ module MobileApp
   class SensorsController < ApplicationController
     def index
       @sensors = Sensor.order(created_at: :asc)
-      @latest_sensor_temperatures = Measurement.last_per_sensor(1).pluck(:sensor_id, :temperature).to_h
+      @latest_sensor_measurements = Measurement.last_per_sensor(1)
+                                               .pluck(:sensor_id, :temperature, :created_at)
+                                               .map { |fields| [
+                                                 fields[0],
+                                                 {
+                                                   :temperature => fields[1],
+                                                   :created_at => fields[2].to_time.to_i,
```

Comment:
I applied your first two suggestions.

I'm not sure what you mean with the third one. A measurement is written once, and that's the timestamp that's relevant to us, isn't it?

Reply:
Rails per convention attaches two different timestamps `created_at` and `updated_at` to any object. On creation, `updated_at` will be the same as `created_at` and since we don't update our records, you're fine. Nevertheless: `updated_at` would reflect the last change to the measurement (incl. creation).